pkg/pillar/scripts/device-steps.sh

```diff
@@ -45,6 +45,25 @@ while [ $# != 0 ]; do
     shift
 done
 
+# Sleep for a bit until /var/run/$1.touch exists
+wait_for_touch() {
+    if [ "$1" = "lisp-ztr" ]; then
+	return
+    fi
+    f=/var/run/"$1".touch
+    waited=0
+    while [ ! -f "$f" ] && [ "$waited" -lt 60 ]; do
+	    echo "$(date -Ins -u) waiting for $f"
+	    sleep 3
+	    waited=$((waited + 3))
```

Comment:
Yetus would complain here. I think the $ can be removed.

Reply:
Actually the $ is needed (in /bin/sh and in bash). No yetus complaints (apart from the tabs which I just fixed)

Reply:
Yes, you are right. I got confused.